.github/workflows/generate.yml

```diff
@@ -44,15 +41,14 @@ jobs:
           key: ${{ hashFiles('src/generate/deno.lock') }}
           path: ${{ env.DENO_DIR }}
 
-      - name: Update Health Files
+      - name: Generate health files
         working-directory: src/generate
         run: ./generate.ts
 
-      - name: Push Changes
-        uses: stefanzweifel/git-auto-commit-action@v4
+      - name: Push changes
+        uses: stefanzweifel/git-auto-commit-action@v5
         if: ${{ github.repository == 'catppuccin/userstyles' && github.ref == 'refs/heads/main' && env.HAS_USERSTYLES_TOKEN }}
         with:
           commit_message: "chore: generate health files"
-          commit_author: "github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>"
```

Comment:
I believe this was added because the default string will include the person who merged the PR as a co-author, and that doesn't really make sense. 

In the words of @nekowinston 
> there's no reason i'm co-authored when i merge a PR in that changes a single line in the readme :baldkek:

Please revert this line :+1:

Reply:
![image](https://github.com/catppuccin/userstyles/assets/47499684/85f3137d-0584-42df-9403-2d511cbca58b)
That explains this!

Reply:
Y'know... [the other action I suggested](https://github.com/EndBug/add-and-commit) doesn't have this problem :D
But yeah for consistency and all that I'll revert it aha.